exercises.html

```diff
@@ -5,7 +5,41 @@
   <title>Quantified Self - Manage Exercises</title>
 </head>
 <body>
+<style>
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+td, th {
+  border: 1px solid #dddddd;
+  text-align: left;
+  padding: 8px;
+}
+
+tr:nth-child(even) {
+  background-color: #dddddd;
+}
+</style>
+
+Enter Your Exercise Below
+<form>
+  Name:<br>
+  <input class='create-exercise-name' type="text" name="name"><br>
+  Calories:<br>
+  <input class='create-calorie-count' type="text" name="calories"></br>
```

Comment:
We'll want to make this page specific since the food page also has create-calorie-count.  We probably want to change this to be more specific on both pages.